nvflare/private/fed/server/sys_cmd.py

```diff
@@ -76,6 +76,14 @@ def get_spec(self):
                     authz_func=self.authorize_client_operation,
                     visible=True,
                 ),
+                CommandSpec(
+                    name="dead",
+                    description="send dead client msg to SJ",
+                    usage="dead <client-name>",
```

Comment:
What's the reason to build an admin command "dead client"?

if so, should we make it consistent for the command format to be "dead client XXX" and "dead client XXX YYY" ?

Reply:
As I said in the description, this command is used to simulate dead client report for the SP, since it's very hard to reproduce it thru connection disrupt. It's only for debugging/testing of dead client handling logic; and is not visible to regular users.

This is only for SP, hence it doesn't make sense to send to clients XXX YYY.